### ISSUE TITLE
cohttp-eio(client): can cohttp-eio handle https connections now?

### ISSUE THREAD

**Issue by RCmerci** [mirage/ocaml-cohttp#951]
I haven't found an example of HTTPS connections using cohttp-eio.

Does cohttp-eio support HTTPS conn now?

**Comment by patricoferris** [comment]
Yes! But you have to do it yourself at the moment, see for example:

https://github.com/geocaml/carbon-intensity/blob/main/src/http_client.ml

(@bikallem or @talex5 might be able to comment more on the proper way to do this :))

**Comment by RCmerci** [comment]
@patricoferris, thanks for the reply!

**Comment by mseri** [comment]
@bikallem can we add an example in the docs or the mdx tests?

**Comment by bikallem** [comment]
Yep, sure can do. I think mdx based README.md seems to have much more edifying effect so I will add it to cohttp-eio folder.
